release_notes.md

```diff
@@ -1,21 +1,10 @@
 # What's New
 
-## Introducing Top SOC Questions
-
-Edit or disable existing prompts or add new prompts based on your security posture.
-
-- You can now customize your interactions with the LLM by providing specific details and guidelines for your chats. Define your SOC questions to customize your LLM interactions.
-
-- Specific modules now have pre-defined questions. For example, under the **Alerts** module, questions include:
-
-    - Get Alert Summary
-    - Generate Response Recommendations
-    - Generate Investigation Report
-    - MITRE ATT&CK Insight
-    
-    Similarly, for the **Incident** module, questions may be:
-    
-    - Get Incident Summary
-    - Generate Response Recommendations
-    - Generate Investigation Report
-    - MITRE ATT&CK Insight
+- The AI Assistant can understand and respond to context sensitive conversations and perform given SOC tasks
+    - Through enhanced comprehension, the bot can navigate to modules, its records, and playbook logs
+    - Sensitive data is being actively masked for all interactions with the API, considering the various modules to which the bot has access
```

Comment:
sensitive data such as: ['URL', 'Email', 'Domain' , 'Host' , 'IP',  'MD5', 'SHA1', 'SHA256']. Please mention it wherever possible.

but this should not be partt of a release note as its already present in the FortiAI from last version.